test/origin_simulator/simulation_test.exs

```diff
@@ -1,58 +1,64 @@
 defmodule OriginSimulator.SimulationTest do
   use ExUnit.Case
 
-  alias OriginSimulator.Simulation
+  import Fixtures
 
-  def test_recipe() do
-    %OriginSimulator.Recipe{origin: "foo",
-                            stages: [%{"at" => 0, "status" => 200, "latency" => "1s"}]}
-  end
-
-  def test_range_recipe() do
-    %OriginSimulator.Recipe{origin: "foo",
-                            stages: [%{"at" => 0, "status" => 200, "latency" => "1s..1200ms"}]}
-  end
-
-  def test_recipe_with_multiple_stages() do
-    %OriginSimulator.Recipe{origin: "foo",
-                            stages: [%{"at" => 0, "status" => 200, "latency" => "0s"},
-                                     %{"at" => "60ms", "status" => 503, "latency" => "1s"}]}
-  end
+  alias OriginSimulator.{Simulation, Recipe}
 
   describe "with loaded recipe" do
     setup do
-      Simulation.add_recipe(:simulation, test_recipe())
+      stages = [%{"at" => 0, "status" => 200, "latency" => "1s"}]
+      recipe = test_recipe(origin: "foo", stages: stages)
+      Simulation.add_recipe(:simulation, recipe)
       Process.sleep(5)
+
+      {:ok, recipe: recipe}
     end
 
     test "state() returns a tuple with http status and latency in ms" do
       assert Simulation.state(:simulation) == {200, 1000}
     end
 
-    test "recipe() returns the loaded recipe" do
-      assert Simulation.recipe(:simulation) == test_recipe()
+    test "recipe() returns the loaded recipe", %{recipe: recipe} do
+      assert Simulation.recipe(:simulation) == recipe
+    end
+
+    test "recipe() returns route", %{recipe: recipe} do
```

Comment:
We don't call recipe on simulation with this one. Maybe it's the wording or my understanding of this.

Reply:
Fixed, it should be `route()`.